QuestionSets/PythonLanguage/ClassesAndObjects/Rajat_Varma/Enemy.py

```diff
@@ -0,0 +1,66 @@
+from Player import *
+
+class Enemy:
+    def __init__(self):
+        self.age = int()
+        self.movement_speed = int()
+        self.health = int()
+        self.attack_power = int()
+
+    def move(self):
+        self.movement_speed += 5
+
+    def stop(self):
+        self.movement_speed = 0
+
+    def take_damage(self):
+        self.health -= 50
+
+    def heal(self):
+        self.health = 100
+
+    def booster(self):
+        self.attack_power += 50
+
+    def attack(self):
+        Player.take_damage(Jason_Brody)

```

Comment:
@rajatvarma Is `Jason_Brody` (love the Far Cry Reference tho) really taking damage ie. is his health decreasing??

Reply:
Yes. If I print his health at the end of Enemy.py, I get 90.

Reply:
man, this code is wrong.... ill tell u why in the next session, just remind me to